### PR TITLE
Story dfcc 412 create new cosmos nuget

### PR DESCRIPTION
**Pull request by IanCrisp**

### CODE REVIEW

File: DFC.Compui.Cosmos/Contracts/IContentPageModel.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+
+namespace DFC.Compui.Cosmos.Contracts
+{
+    public interface IContentPageModel : IDocumentModel
+    {
+        string? CanonicalName { get; set; }
+
+        public IList<string>? AlternativeNames { get; set; }
```
Comment (snipered2003) [inline DFC.Compui.Cosmos/Contracts/IContentPageModel.cs:9]: public not needed.

File: DFC.Compui.Cosmos/DFC.Compui.Cosmos.csproj [hunk 2]
```diff
@@ -1,6 +1,40 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```
Comment (snipered2003) [inline DFC.Compui.Cosmos/DFC.Compui.Cosmos.csproj:3]: Is this correct, your switching back to standard 2.1?

Comment (IanCrisp) [reply DFC.Compui.Cosmos/DFC.Compui.Cosmos.csproj:3]: Yes this is correct. It is a Nuget project

File: DFC.Compui.Cosmos/Contracts/IContentPageModel.cs (hunk 1 above)
Comment (IanCrisp) [reply DFC.Compui.Cosmos/Contracts/IContentPageModel.cs]: Removed

**Pull request merged**